### DISCUSSION TITLE
changing total body water / fat / protein content

### DISCUSSION THREAD

**Discussion by Fate-afshar** [Open-Systems-Pharmacology/Forum#1237] [How-Tos]
hi!
I'm working on a pbpk model for a subpopulation. and I need to increase /decrease the total body water, total body fat, and total body protein level.  I was wondering how to do this? I presume I need mobi for this. there is a fractional value but I need the total value to change.

**Comment by Scorrt542** [comment]
Hello, have you solved this problem？

**Comment by Fate-afshar** [comment]
Hi , this project is from a while back so my memory may be a bit hazy. We
weren't able to change the whole fat or protein content but I think I was
able to alter the water content and since it didn't make a difference so
the idea was abandoned altogether. They may have added this ability to
newer versions. I'll keep you posted if i find out
